draft-ietf-quic-qpack.md

```diff
@@ -134,7 +134,7 @@ HTTP fields:
 
 HTTP field line:
 
-: A name-value pair sent as part of an HTTP field section.  See Section 5 of
+: A name-value pair sent as part of an HTTP field section.  See Section 5.4 of
```

Comment:
Since 5.4 is specifically headers (and not trailers), we probably need to reference both 5.4 and 5.6.  HTTP had a similar change.